curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-a-gradual-css-linear-gradient.md

```diff
@@ -10,7 +10,7 @@ forumTopicId: 301047
 <section id='description'>
 Applying a color on HTML elements is not limited to one flat hue. CSS provides the ability to use color transitions, otherwise known as gradients, on elements. This is accessed through the <code>background</code> property's <code>linear-gradient()</code> function. Here is the general syntax:
 <code>background: linear-gradient(gradient_direction, color 1, color 2, color 3, ...);</code>
-The first argument specifies the direction from which color transition starts - it can be stated as a degree, where 90deg makes a vertical gradient and 45deg is angled like a backslash. The following arguments specify the order of colors used in the gradient.
+The first argument specifies the direction from which color transition starts - it can be stated as a degree, where 90deg makes a gradient from left to right and 45deg is angled like a backslash. The following arguments specify the order of colors used in the gradient.
```

Comment:
While we're already making changes, we might as well tidy this small bit up as well.

```suggestion
The first argument specifies the direction from which color transition starts - it can be stated as a degree, where <code>90deg</code> makes a gradient from left to right and <code>45deg</code> is angled like a backslash. The following arguments specify the order of colors used in the gradient.
```